testutil/compose/define.go

```diff
@@ -64,16 +88,16 @@ func Define(ctx context.Context, dir string, clean bool, seed int) error {
 		conf.Def.Operators[i].Address = ""
 	}
 
-	for i, key := range p2pkeys {
-		// Best effort creation of folder, rather fail when saving p2pkey file next.
-		_ = os.MkdirAll(nodeFile(dir, i, ""), 0o755)
+	return conf, p2pkeys
+}
 
-		err := crypto.SaveECDSA(nodeFile(dir, i, "p2pkey"), key)
-		if err != nil {
-			return errors.Wrap(err, "save p2pkey")
-		}
-	}
+// nodeFile returns the path to a file in a node folder.
+func nodeFile(dir string, i int, file string) string {
```

Comment:
why a separate function for just one line?

Reply:
easier to read, and used in many places